### ISSUE TITLE
LiteralProto max size

### ISSUE THREAD

**Issue by ailzhang** [pytorch/xla#866]
I got error from running a test with large tensor. 
```
> /scratch/ailzhang/pytorch/test/common_utils.py(590)assertTensorsEqual()
-> b = b.to(a)
(Pdb) n
[libprotobuf ERROR external/protobuf_archive/src/google/protobuf/message_lite.cc:447] xla.LiteralProto exceeded maximum protobuf size of 2GB: 2200000032
2019-07-30 21:43:47.485270: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Invalid argument: LiteralProto has no shape
zsh: abort (core dumped)  python test_cuda.py TestCuda.test_DoubleTensor_mean_64bit_indexing
```
**This is not a repro**
The test is essentially running `mean` test below, however I couldn't repro the error when running the standalone script, but only in test. 
```
import torch
import torch_xla

device = 'xla:0'

M = torch.ones(275000000).double()
cpu_result = torch.mean(M)
xla_result = torch.mean(M.to(device))
import pdb
pdb.set_trace()
xla_result.to(cpu_result)
```
If the info above is not enough to locate the issue, let me know, maybe I'm missing sth to trigger it without test suite.

**Comment by ailzhang** [comment]
I'll remove the label until we can find a reliable repro.

**Comment by dlibenzi** [comment]
It is a known limit of Protocol Buffers the inability to serialize over 2GB, and since the GRPC Interface to the TF/XRT uses PB at this time, there is nothing we can do:

https://github.com/protocolbuffers/protobuf/blob/912d8195dad66d158fed2c9dc38a039448f3f43e/src/google/protobuf/message_lite.cc#L347
